### ISSUE TITLE
Formatting works in "Read-Only" mode, but doesn't save.

### ISSUE THREAD

**Issue by orblivion** [kentonv/etherpad-lite#43]
* Make an Etherpad
* Make sure there's some text
* Make a read-only share link
* Open in incognito tab
* Select text
* Bold the text

It will turn bold. However if you refresh the page, the formatting will reset. So, it doesn't save.

**Comment by kentonv** [comment]
It's likely that this is an upstream bug, as the Sandstorm port's permissions merely enable the read-only support that exists in the upstream code.

**Comment by orblivion** [comment]
Are you running the latest version? I don't want to report something that's already fixed.

**Comment by orblivion** [comment]
(I mean, are you using the latest version for this package.)

**Comment by kentonv** [comment]
Yes, Etherpad on Sandstorm is based on Etherpad 1.6.1, which appears still to be the most-recent release, although there's been a number of commits to master since then.

**Comment by kentonv** [comment]
That said I do think it makes sense to test upstream to see if the bug is there before reporting.

TBH I am not entirely sure how one shares a pad read-only using stock etherpad.
